app/services/cap_update_service.rb

```diff
@@ -0,0 +1,34 @@
+class CapUpdateService
```

Comment:
From its name, it sounds like this class should be generalisable to comms caps but isn't currently, since it's currently assuming `device_type: 'std_device'` inside `update_cap!`. How about passing in `device_type` via the initializer?

Reply:
Sure, good idea 👍